packages/devops/scripts/lambda/helpers/creation.py

```diff
@@ -157,3 +167,43 @@ def create_instance(
         setup_subdomains_via_gateway(deployment_type, new_instance_object, subdomains_via_gateway, deployment_name)
 
     return new_instance_object
+
+def create_db_instance_from_snapshot(
+  instance_name,
```

Comment:
```suggestion
  deployment_name,
  deployment_type,
```
Although we're not using RDS for anything other than Tupaia atm, I feel safer bringing in the deployment type stuff so that it's easily distinguished if we do (actual example: we may be setting up the EMIS database for Laos in RDS)